File: src/main/resources/output/IFC4X3_csharp/Entities/IfcExternalInformation/IfcClassification/IfcClassification.cs
public class IfcClassification : IfcExternalInformation
{
    public IfcLabel Source { get; set; }
    public IfcLabel Edition { get; set; }
    public IfcDate EditionDate { get; set; }
    public IfcLabel Name { get; set; }
    public IfcText Description { get; set; }
    public IfcURIReference Specification { get; set; }
    public List<IfcIdentifier> ReferenceTokens { get; set; }

    // === INVERSE CLAUSES ===
    // ClassificationForObjects : SET [0:?] OF IfcRelAssociatesClassification FOR RelatingClassification
    // HasReferences : SET [0:?] OF IfcClassificationReference FOR ReferencedSource
}

File: src/main/resources/output/IFC4X3_vbnet/Entities/IfcExternalInformation/IfcClassification/IfcClassification.vb
Public Class IfcClassification Inherits IfcExternalInformation
    Public Source As IfcLabel
    Public Edition As IfcLabel
    Public EditionDate As IfcDate
    Public Name As IfcLabel
    Public Description As IfcText
    Public Specification As IfcURIReference
    Public ReferenceTokens As List(Of IfcIdentifier)

    ' === INVERSE CLAUSES ===
    ' ClassificationForObjects : SET [0:?] OF IfcRelAssociatesClassification FOR RelatingClassification
    ' HasReferences : SET [0:?] OF IfcClassificationReference FOR ReferencedSource
End Class
